tests/compiler_test.py

```diff
@@ -23,13 +23,18 @@ def loadUFO(filename):
 class CompilerTest(unittest.TestCase):
     _tempdir, _num_tempfiles = None, 0
 
-    def testTestFont(self):
+    def test_TestFont(self):
         # We have specific unit tests for CFF vs TrueType output, but we run
         # an integration test here to make sure things work end-to-end.
         # No need to test both formats for every single test case.
         self.expectTTX(compileTTF(loadUFO("TestFont.ufo")), "TestFont.ttx")
         self.expectTTX(compileOTF(loadUFO("TestFont.ufo")), "TestFont-CFF.ttx")
 
+    def test_regressions(self):
```

Comment:
from a quick glance to the test method name "test_regressions" it's not clear to me what this test is about? Bug108 is also not very telling. I have to read at the test data itself or find the issue on github to understand what it's about. Maybe rename the test method to be more specific, or add a docstring to it?